kratos/tests/cpp_tests/global_pointers/test_global_pointers_vector.cpp

```diff
@@ -46,8 +47,38 @@ KRATOS_TEST_CASE_IN_SUITE(GlobalPointersContainerTest, KratosCoreFastSuite)
     {
         KRATOS_CHECK_EQUAL(&new_global_pointers[i], &global_pointers_container[i]);
     }
-    
+}
+
+KRATOS_TEST_CASE_IN_SUITE(GlobalPointersContainerInVariableTest, KratosCoreFastSuite)
+{
+    Model current_model;
+    Model loaded_model;
+    ModelPart& mp = current_model.CreateModelPart("test");
+    mp.AddNodalSolutionStepVariable(TEMPERATURE); //not to have an empty var list
+
+    const auto & node_1 = mp.CreateNewNode(1,1.0,2.0,3.0);
```

Comment:
Put the & in the middle is the most unholy thing
![imagen](https://user-images.githubusercontent.com/19991680/35767644-8a9c6856-08f0-11e8-9944-e72c6c1db7b3.png)

Reply:
Weeeeeell I've move it... (But I still prefer to put it in the middle when passing arguments)